packages/edit-site/src/components/site-hub/index.js

```diff
@@ -50,7 +50,7 @@ const SiteHub = memo( ( { isTransparent, className } ) => {
 				dashboardLink:
 					getSettings().__experimentalDashboardLink || 'index.php',
 				homeUrl: getUnstableBase()?.home,
-				siteTitle: getSite()?.title,
+				siteTitle: getEntityRecord( 'root', 'site' )?.title,
```

Comment:
Isn't this a breaking change? I guess we need to manually define `getSite` now?

Reply:
Nvm, just saw the comment above.

Reply:
This is a temporary change. I will resolve this before the PR is ready to be merged. I am tracking it here: https://github.com/WordPress/gutenberg/pull/59243#discussion_r1505151487.